### ISSUE TITLE
Popover, Tooltip, and probably more components need `aria-controls`

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2253]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

The global aria-controls property identifies the element (or elements) whose contents or presence are controlled by the element on which this attribute is set.

Such elements should have `aria-controls` set to a value that refers to the element that serves as the popup. 

The aria-controls only needs to be set when the popup is visible, but it is valid and easier to program to reference an element that is not visible.

**Comment by Kiarokh** [comment]
most are fixed I believe
